contents/handbook/engineering/product-design.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Product Design, for Engineers
+sidebar: Handbook
+showTitle: true
+---
+
+It is up to you when to involve our product designers in your work.
+
+## v0.1 or v2?
+
+As the feature owner, you should make a choice if you're building a very basic first iteration of something, or if you're improving the experience.
+
+If you're improving an existing feature that is popular, you are probably creating v2.
+
+If you're shipping an entirely new feature (i.e. SQL for PostHog), then you should figure out if anyone cares, which usually means creating a v0.1, and releasing it behind a feature flag to some friendly users (pro tip: make friends by being support hero).
+
+Please _communicate_ to product design what your expectations are!
```

Comment:
Can we clarify what are the requirements for the quality of the design in v0.1 and v2? 

Can we ever release something as a v2 without both an engineer's and a designer's ✅, especially if it involves UI changes?

I'd argue that if we want to keep a consistently high quality of product, a v2 or unflagged user facing feature should have the designer's blessing. Luckily they're easy to come by if you use standard components in familiar patterns, which are getting properly documented more and more.

Reply:
Sometimes it might be worth looping into a designer at the very start to explore what it could be such as doing concepting work in deciding what the feature/product could be.